src/providers/walletconnectNotify.ts

```diff
@@ -89,10 +89,22 @@ function formatMessage(event: Event, proposal) {
 }
 
 export async function send(event: Event, proposal, addresses: string[]) {
+  if (!isConfigured()) {
+    console.log('[WalletConnect] Sending skipped: client not setup');
+  }
```

Comment:
Better to check this outside of the function,
```typescript
const isConfigured = WALLETCONNECT_NOTIFY_SERVER_URL &&
    WALLETCONNECT_PROJECT_SECRET &&
    WALLETCONNECT_PROJECT_ID &&
    WALLETCONNECT_NOTIFICATION_TYPE
```

and then 
```suggestion
  if (!isConfigured) {
    console.log('[WalletConnect] Sending skipped: client not setup');
  }
```